custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
```

Comment:
@esoergel I have done a quick test to see how numbers add up. And it looks right. Please have a closer look at this to make sure it looks good to you.